### ISSUE TITLE
English Dub config for other Languages

### ISSUE THREAD

**Issue by diegorocha2007** [Rinnegatamante/ff4_vita#1]
As of now, the english dub is activated by default in english, french, german, italian and russian. With the configurator working to change to japanese dub if so desired.

However, Japanese, Spanish, and Portuguese languages defaults to the japanese dub, with no option to choose english dub in this case.

I wonder if an option in the configurator can be added to force the english dub in these languages as well.

Also, the Spanish language text is broken, it shows korean characters in menu, and blank ones in subtitles, it might be worth checking out.

Thanks for the hard work!

**Comment by Rinnegatamante** [comment]
The spanish text should be fixed with v.1.1. Can you confirm?
For the english dub option, that's a nice suggestion. I'll implement it in a next update.

**Comment by diegorocha2007** [comment]
Yes, the Spanish text is fixed on v1.1, I just confirmed it on my Vita.
Thanks for accepting my suggestion, I'll wait for the next update to start playing with English dub.

**Comment by Rinnegatamante** [comment]
English dub forcing added with: https://github.com/Rinnegatamante/ff4_vita/commit/24a5d9235624d0ba99fd3141f122b6e6cbe715ee
Will be available in 1.2

**Comment by diegorocha2007** [comment]
Oh, thanks. Looking forward to it.

**Comment by diegorocha2007** [comment]
Hey Rinne, would you mind releasing the 1.2 update with the dub forcing option? I'm still looking forward to playing with this change.

Thanks in advance!

**Comment by Rinnegatamante** [comment]
Pretty busy with university for a couple of weeks still. Before February ends i'll release it for sure. Totally forgot about it so thanks for reminding me.

**Comment by diegorocha2007** [comment]
No problem man, real life calls. I'll wait.
